tools/benchmark

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+LEGOFY=$(pwd)/bin/legofy;
+PATH_IMAGES=$(pwd)/images/benchmark/;
+
+benchmark() {
+  local flags=$@;
+
+  # Default arguments
+  if [[  $flags = '' ]]; then
+    flags='--filter-quality=80 --pattern-quality=80 --quality=good'
+  fi
+
+  # Loop for each images
+  for img in $(ls $PATH_IMAGES)
```

Comment:
[Bad practice to use `ls` in a script.](http://mywiki.wooledge.org/ParsingLs).

```sh
for img in $PATH_IMAGES/*
```

Simpler and safe.